ext/strscan/extconf.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 require 'mkmf'
-$INCFLAGS << " -I$(top_srcdir)"
+$INCFLAGS << " -I$(top_srcdir) -fPCI"
```

Comment:
Typo?

```diff
- $INCFLAGS << " -I$(top_srcdir) -fPCI"
+ $INCFLAGS << " -I$(top_srcdir) -fPIC"
```

Reply:
I did mean that but the strange thing is that either version allows compilation on my machine O_O

I'll push a new commit.